app/Services/BaseService.php

```diff
@@ -15,14 +16,21 @@ abstract class BaseService
     use WritesToConsole;
 
     protected static $category;
+
     protected static $displayName;
 
     protected $organization = 'library'; // Official repositories use `library` as the organization name.
+
```

Comment:
This division is unnecessary for the code style we use on Takeout. Can we keep it as it currently is on this and all the other files?

Reply:
Fixed it manually. I'll try to add Duster to it later.